components/QrCodeRender.tsx

```diff
@@ -1,18 +1,22 @@
+import { useContext, useEffect, useState } from 'react';
 import qrCode from '../utils/QrCode';
 import {TInitialValuesRegister} from '../components/FormQR/SchemaFormQR';
+import QRSizeContext from "../context/Context";
 
 type TDataQRcode = {
     dataQRcode: TInitialValuesRegister | any;
 }
 
-const QRCodeRender = ({dataQRcode}: TDataQRcode) => (
-    <>
-        <div dangerouslySetInnerHTML={{__html: qrCode.createVCardQr(dataQRcode, {typeNumber: 4, cellSize: 6})}}>
-        </div>
-        <div dangerouslySetInnerHTML={{__html: qrCode.createVCard(dataQRcode)}}>
-        </div>
-    </>
-  );
+const QRCodeRender = ({dataQRcode}: TDataQRcode) => {
+    const { example_size } = useContext(QRSizeContext);
+
+    return (
+        <>
+            <div>Here is your QR Code!</div>
+            <div dangerouslySetInnerHTML={{__html: qrCode.createVCardQr(dataQRcode, {typeNumber: 5, cellSize: example_size})}}></div>
```

Comment:
where this `qrCode` is coming from? Is it some global library available in html?

Reply:
main script holds in `QrCode.js`, its library that we are using for QR code generation.

Reply:
Actually, it's not a library, just in file QrCode.js we are using the immediately called function, this is not the best solution, it can be rewritten to a normal function and then we can don’t use a special attribute from Next.js documentation as 'dangerouslySetInnerHTML' https://nextjs.org/docs/basic-features/script 
If you want, you can do it. Take your time🙂